components/crossplane-config/staging/kustomization.yaml

```diff
@@ -3,3 +3,4 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
   - ../base
+  - provider-config.yaml
```

Comment:
The ProviderConfig shouldn't reside in the `crossplane-config` component since that's deployed to the target (EaaS) clusters. We need this on the "member" clusters where the control plane is running (the `crossplane-control-plane` component).